lib/web/apiserver.go

```diff
@@ -2355,6 +2360,132 @@ func createIdentityContext(login string, sessionCtx *SessionContext) (srv.Identi
 	}, nil
 }
 
+type UILock struct {
+	Name      string           `json:"name"`
+	Message   string           `json:"message"`
+	Expires   string           `json:"expires"`
+	CreatedAt string           `json:"createdAt"`
+	CreatedBy string           `json:"createdBy"`
+	Targets   types.LockTarget `json:"targets"`
+}
+
+func (h *Handler) getClusterLocks(
+	w http.ResponseWriter,
+	r *http.Request,
+	p httprouter.Params,
+	sessionCtx *SessionContext,
+	site reversetunnel.RemoteSite) (interface{}, error) {
+	ctx := r.Context()
+	clt, err := sessionCtx.GetUserClient(r.Context(), site)
```

Comment:
```suggestion
	clt, err := sessionCtx.GetUserClient(ctx, site)
```

Use `ctx` or `r.Context()` consistently. Switching back and forth between the two is confusing.